modules/nf-core/gunzip/main.nf

```diff
@@ -21,10 +21,11 @@ process GUNZIP {
     def args = task.ext.args ?: ''
     gunzip = archive.toString() - '.gz'
     """
-    gunzip \\
-        -f \\
+    gzip \\
```

Comment:
```suggestion
    # Not calling gunzip itself because it creates files
    # with the original group ownership rather than the
    # default one for that user / the work directory
    gzip \\
```